webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -181,12 +181,40 @@ class RudderCRunner(
     } yield ()
   }
 
+  // Create an empty reporting bundle for compatibility with the fallback process
+  // Can be removed once rudderc is the only generation method
+  def emptyReportingFile(technique : EditorTechnique) = {
+
+    val bundleParams = if (technique.parameters.nonEmpty) technique.parameters.map(_.name.canonify).mkString("(",",",")") else ""
+    val content =
+      s"""bundle agent ${technique.bundleName.value}_rudder_reporting${bundleParams}
+         |{
+         |}"""
+
+    val reportingFile = File(outputPath) / "rudder_reporting.cf"
+
+
+    TechniqueWriterLoggerPure.error(s"PLOUF target '${technique.name}' in path ${reportingFile.path.toString}")
+
+    IOResult.effect(s"Could not write empty reporting Technique file '${technique.name}' in path ${reportingFile.path.toString}") {
+      reportingFile.createFileIfNotExists(true).write(content.stripMargin('|'))
```

Comment:
by default, `stripMargin` uses `|` , you can remove it (but it's ok to keep it too)